### PR TITLE
add a contributing style-guide document

### PR DESCRIPTION
**Pull request by igalic**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+* Please ensure a (single) empty line before and after headers
+* Each sentences should be put on its own line
+
+## Glossary
+
+The word "Jail" (as concept) should always be written as with upper-case J.
+The word "jail" (command or system function) should always be written as `jail`.
+To distinguish between the command and the function you can use the man page section: e.g.: `jail(8)`
+
+## CoC
```
Comment (igalic) [inline CONTRIBUTING.md:18]: missing?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+In order to contribute to this Handbook please be aware of the following style guide:
+
+## Markdown
+
+* We are using GitHub flavoured Markdown.
+* The biggest header in chapters should be `h3`, or `###`
+* Please ensure a (single) empty line before and after headers
+* Each sentences should be put on its own line
+
+## Glossary
```
Comment (igalic) [inline CONTRIBUTING.md:12]: better title

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+The word "Jail" (as concept) should always be written as with upper-case J.
+The word "jail" (command or system function) should always be written as `jail`.
+To distinguish between the command and the function you can use the man page section: e.g.: `jail(8)`
+
```
Comment (igalic) [inline CONTRIBUTING.md:17]: add 'you' / 'we' style explanation

**Review by gronke — APPROVED** [review]

**Review by igalic — COMMENTED** [review]
needs to be betterer

**Comment by gronke** [comment]
We can enhance the document as we go.

**Pull request merged**